cmd/cogito/main_test.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
```

Comment:
I am finally introducing an assert library, after having tried _many_.
This is quite nice and allows to reduce the verbosity, focusing on the intent.